src/components/Receive/AddressVerifyModal.js

```diff
@@ -56,7 +57,7 @@ const AddressVerifyModal = ({
       </Text>
       <View style={styles.addressDetailsView}>
         <Text secondary style={styles.paragraph}>
-          {address}
+          {addressInfo.address}
```

Comment:
Ledger devices show the staking key path in the verification flow, so it's something you should probably consider adding.